corehq/apps/integration/payments/tables.py

```diff
@@ -64,7 +71,10 @@ def render_verify_select(self, record, value):
         }
         # All columns are required except the checkbox
         required_fields = list(self.base_columns.keys())
-        required_fields.remove('verify_select')
+
+        for optional_field in self.OPTIONAL_FIELDS:
+            if optional_field in required_fields:
+                required_fields.remove(optional_field)
```

Comment:
Nit: This could be simplified to:
```suggestion
        required_fields = set(required_fields) - set(optional_fields)
```